pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go

```diff
@@ -28,14 +28,19 @@ import (
 )
 
 const (
-	MpiPluginName = "mpi"
-	DefaultPort   = 22
+	// MPIPluginName is the name of the plugin
```

Comment:
Please seperate the mpi related refactor to another pr. It's inconsist with title.

Reply:
The update comes from the CI check - Code Verify. If not included in this PR, this CI item will fail.

Reply:
OK, I see.